_templates/community.html

```diff
@@ -134,6 +134,7 @@ <h3 id="ireland"><img src="/img/flags/IE.png" alt="Irish flag">Ireland</h3>
     <h3 id="israel"><img src="/img/flags/IL.png" alt="Israeli flag">Israel</h3>
     <p>
       <a href="http://www.bitcoin.org.il/">איגוד הביטקוין הישראלי</a>
```

Comment:
A line break is missing at this end of this line. It needs to be in place to accommodate the additional entry.